api/common/config.py.example

```diff
@@ -13,7 +13,7 @@ config = { \
         'db_name': '',
         'db_user': '',
         'db_password': '',
-        'smtp_host': '',
+        'smtp_host': 'smtp.gmail.com',
```

Comment:
i presume this is debug code that was pushed?

Reply:
Thanks for pointing that out! I thought to use it for the smtp config setup. Thanks!